src/internal/operators/delayWhen.ts

```diff
@@ -1,15 +1,24 @@
-import { Operator } from '../Operator';
+/** @prettier */
 import { Subscriber } from '../Subscriber';
 import { Observable } from '../Observable';
-import { Subscription } from '../Subscription';
-import { ComplexOuterSubscriber, ComplexInnerSubscriber, innerSubscribe } from '../innerSubscribe';
-import { MonoTypeOperatorFunction, TeardownLogic } from '../types';
+import { MonoTypeOperatorFunction } from '../types';
 import { lift } from '../util/lift';
+import { OperatorSubscriber } from './OperatorSubscriber';
+import { concat } from '../observable/concat';
+import { take } from './take';
```

Comment:
issue: This `take` import isn't used. We really should re-enable the `unused-import` lint rule.

The `concat` and `ignoreElements` imports aren't used, either.

Reply:
It actually is used for the deprecated path. :) But you're right, I think we need to reenable that.